File: solutions/cpp/two-fer/1/two_fer.cpp
#include "two_fer.h"
#include <string>
using namespace std;

namespace two_fer
{
    string two_fer(string name)
    {
        return "One for " + name + ", one for me.";        
    }
    string two_fer(){
        return two_fer("you");
    }
} // namespace two_fer

File: solutions/java/two-fer/1/src/main/java/Twofer.java
public class Twofer {
    public String twofer(String name) {
        String ans;
        if(null==name){
            ans = "One for you, one for me.";
        } else {
            ans =  "One for " +name +", one for me.";
        }
        return ans;
    }
}
